test-support/helpers/x-select.js

```diff
@@ -9,25 +9,25 @@ import jQuery from 'jquery';
  * @param {string} texts - text of the option you are picking
  */
 export function select(selector, ...texts) {
-  let $select = selector instanceof jQuery ? selector : Ember.$(selector);
-  let $options = $select.find(`option`);
+  const $select = selector instanceof jQuery ? selector : Ember.$(selector);
+  const $options = $select.find(`option`);
```

Comment:
The real fix is below, but I looked through the rest of the helper and found no variables that were being set again. To make this clear I changed everything to `const`.